src/lib/tooltip/README.md

```diff
@@ -1,24 +1,28 @@
 # MdTooltip
-Tooltip allows the user to specify text to be displayed when the mouse hover over an element.
+Tooltip allows the user to specify text to be displayed when the mouse hovers over an element.
+
+The positions `start` and `end` should be used instead of `left` and `right` respectively when the tooltip positioning should change depending on the HTML dir global attribute for left-to-right or right-to-left layout.
 
 ### Examples
 A button with a tooltip
 ```html
-<button md-tooltip="some message" tooltip-position="below">Button</button>
+<button md-tooltip="some message" tooltip-position="bottom">Button</button>
 ```
 
+
+
 ## `[md-tooltip]`
 ### Properties
 
 | Name | Type | Description |
 | --- | --- | --- |
 | `md-tooltip` | `string` | The message to be displayed. |
-| `tooltip-position` | `"above"|"below"|"before"|"after"` | The position of the tooltip. |
+| `tooltip-position` | `"start"|"end"|"top"|"bottom"|"left"|"right"` | The position of the tooltip. |
```

Comment:
We discussed this in the past and had decided that tooltip would use `before` and `after` instead of `start` and `end`. 

The reasoning was that anything that is positioned _relative to some other element_ would use `before` and `after`, and anything where the alignment is global (just based on the page layout) will be `start` and `end`.